components/properties/src/provider.rs

```diff
@@ -579,51 +404,64 @@ macro_rules! expand {
                     const KEY: DataKey = data_key!(concat!("props/", $enum_s, "@1"));
                 }
 
-                #[cfg(feature = "datagen")]
-                impl Default for $code_point_map_marker {
-                    fn default() -> Self {
-                        Self
-                    }
+
+                #[doc = core::concat!("Data marker for parsing the names of the values of the '", $enum_s, "' Unicode property")]
+                #[derive(Debug, Default)]
+                #[cfg_attr(
+                    feature = "datagen",
+                    derive(databake::Bake),
+                    databake(path = icu_properties::provider),
+                )]
+                pub struct $name_value_marker;
+
+                impl DataMarker for $name_value_marker {
+                    type Yokeable = names::PropertyValueNameToEnumMapV1<'static>;
                 }
 
-                #[cfg(feature = "datagen")]
-                impl databake::Bake for $code_point_map_marker {
-                    fn bake(&self, env: &databake::CrateEnv) -> databake::TokenStream {
-                        env.insert("icu_properties");
-                        databake::quote!{ ::icu_properties::provider::$code_point_map_marker }
-                    }
+                impl KeyedDataMarker for $name_value_marker {
+                    const KEY: DataKey = data_key!(concat!("props/names/", $enum_s, "@1"));
                 }
 
-                #[doc = core::concat!("Data marker for the names of the values of the '", $enum_s, "' Unicode property")]
-                #[derive(Debug)]
-                pub struct $value_name_marker;
 
-                impl DataMarker for $value_name_marker {
-                    type Yokeable = PropertyValueNameToEnumMapV1<'static>;
+                #[doc = core::concat!("Data marker for producing short names of the values of the '", $enum_s, "' Unicode property")]
+                #[derive(Debug, Default)]
+                #[cfg_attr(
+                    feature = "datagen",
+                    derive(databake::Bake),
+                    databake(path = icu_properties::provider),
+                )]
+                pub struct $value_short_name_marker;
+
+                impl DataMarker for $value_short_name_marker {
+                    type Yokeable = names::PropertyEnumToValueNameMapV1<'static>;
                 }
 
-                impl KeyedDataMarker for $value_name_marker {
-                    const KEY: DataKey = data_key!(concat!("props/names/", $enum_s, "@1"));
+                impl KeyedDataMarker for $value_short_name_marker {
+                    const KEY: DataKey = data_key!(concat!("props/names/to/short/", $enum_s, "@1"));
```

Comment:
q: is this name good? Should I rename the NameToValue one to also be `props/names/from`?